CHANGELOG.md

```diff
@@ -8,4 +12,4 @@
 
 # 1.0.0 - 2014-10-04
 
-✨ Initial release from [postcss-color](https://github.com/postcss/postcss-color)
```

Comment:
Not in your browser maybe, but in good browsers they are.

![screen shot 2015-06-13 at 18 38 28](https://cloud.githubusercontent.com/assets/157534/8145125/7ce9dc82-11fb-11e5-8458-5ddfb48357e4.png)
![screen shot 2015-06-13 at 18 38 40](https://cloud.githubusercontent.com/assets/157534/8145126/7cefe0d2-11fb-11e5-8868-50caa8e05622.png)